modules/common/kv_db/kv_db.cc

```diff
@@ -86,7 +86,8 @@ bool KVDB::Has(const std::string &key) {
 
   std::string value;
   const auto status = GetDB()->Get(options, key, &value);
-  return !status.IsNotFound();
+  CHECK(status.ok() || status.IsNotFound()) << status.ToString();
```

Comment:
No need to CHECK. Even no need to show error.
Both Found and NotFound should be normal for a function named "Has()".